docker_test.go

```diff
@@ -1749,6 +1749,40 @@ func TestDockerContainerCopyFileToContainer(t *testing.T) {
 	}
 }
 
+func TestDockerCreateContainerWithFiles(t *testing.T) {
+	ctx := context.Background()
+	hostFileName := "./testresources/hello.sh"
+	copiedFileName := "/hello_copy.sh"
+
+	nginxC, err := GenericContainer(ctx, GenericContainerRequest{
+		ContainerRequest: ContainerRequest{
+			Image:        "nginx:1.17.6",
+			ExposedPorts: []string{"80/tcp"},
+			WaitingFor:   wait.ForListeningPort("80/tcp"),
+			Files: []ContainerFile{
+				{
+					HostFilePath:      hostFileName,
+					ContainerFilePath: copiedFileName,
+					FileMode:          700,
```

Comment:
What happens if a developer misses the FileMode? Following the code, it will invoke the `CopyToContainer` method with a default `0` value as FileMode, and I think it will cause a failure while packing the file into the Tar file.

I wonder if we could at least add a validation for a valid FileMode 🤔

Reply:
@mdelapenya I tried to run `CopyToContainer` with `0` FileMode and didn't get any errors :/

Reply:
@mdelapenya `ls -l` inside container:
```bash
----------   1 root root   69 Jan  1  1970 hello_copy.sh
drwxr-xr-x   2 root root 4096 Nov 10  2019 home
drwxr-xr-x   1 root root 4096 Dec 28  2019 lib
```